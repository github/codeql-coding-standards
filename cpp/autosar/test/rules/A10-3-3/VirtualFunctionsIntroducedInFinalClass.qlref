rules/A10-3-3/VirtualFunctionsIntroducedInFinalClass.ql