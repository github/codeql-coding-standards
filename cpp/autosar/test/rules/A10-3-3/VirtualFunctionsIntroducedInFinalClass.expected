| test.cpp:12:8:12:8 | g | Non-final virtual function $@ is introduced in final class $@. | test.cpp:12:8:12:8 | g | g | test.cpp:9:7:9:13 | Derived | Derived |
| test.cpp:14:16:14:16 | h | Non-final virtual function $@ is introduced in final class $@. | test.cpp:14:16:14:16 | h | h | test.cpp:9:7:9:13 | Derived | Derived |
